thetanuts/thetanuts_test.py

```diff
@@ -130,7 +132,7 @@ def get_env(variable: str) -> str:
 # Maker checks contract for offer info
 offer = thetanuts.get_offer_details(
     contract_address=bridge_contract_address,
-    offer_id=int(vault_contract_address, 16),
+    offer_id=vault_id << 16 + vault_epoch,
```

Comment:
Is this expression correct?

Due to operators precedence this is equivalent to:

`vault_id << (16 + vault_epoch)`

`vault_id` is currently zero, as returned from:
`vault_id = bridgeContract.functions.vaultIndexToAddress(vault_contract_address).call()`

So the result of this expression is always zero despite of `vault_epoch` (currently 69 + 1).
Is `vault_id` expected to change?

Reply:
oh wow my bad, alws assume bitshift has precedence. will add the brackets!

vault_id is always fixed. vault_epoch will monotonically increase!

Reply:
Thank you!